README.md

```diff
@@ -102,503 +102,77 @@ identifiable. If you are prototyping, you can use `java.util.UUID` to generate a
-
-while (historicalMessages.hasNext()) {
-    historicalMessages = historicalMessages.next()
-    println(historicalMessages.items.toString())
-}
-
-println("End of messages")
-```
-
-## Online status
-
-### Retrieving online members
-
-You can get the complete list of currently online or present members, their state and data, by calling the `presence#get` method.
-
-```kotlin
-// Retrieve the entire list of present members
-val presentMembers = room.presence.get()
-
-// You can supply a clientId to retrieve the presence of a specific member with the given clientId
-val presentMember = room.presence.get(clientId = "client-id")
-
-// You can call this to get a simple boolean value of whether a member is present or not
-val isPresent = room.presence.isUserPresent("client-id")
-```
-
-Calls to `presence#get()` will return a list of the presence messages, where each message contains the most recent
-data for a member.
-
-### Entering the presence set
-
-To appear online for other users, you can enter the presence set of a chat room. While entering presence, you can provide optional data that
-will be associated with the presence message.
-
-```kotlin
-room.presence.enter(
-    JsonObject().apply {
-        addProperty("status", "online")
-    },
-)
-```
-
-### Updating the presence data
-
-Updates allow you to make changes to the custom data associated with a present user. Common use-cases include updating the users'
-status or profile picture.
-
-```kotlin
-room.presence.update(
-    JsonObject().apply {
-        addProperty("status", "busy")
-    },
-)
-```
-
-### Leaving the presence set
-
-Ably automatically triggers a presence leave if a client goes offline. But you can also manually leave the presence set as a result of a UI
-action. While leaving presence, you can provide optional data that will be associated with the presence message.
-
-```kotlin
-room.presence.leave(
-    JsonObject().apply {
-        addProperty("status", "Be back later!")
-    },
-)
-```
-
-### Subscribing to presence updates
-
-You can provide a single listener, if so, the listener will be subscribed to receive all presence event types.
+Start by creating a new Android project and installing the Chat SDK using the instructions described above. Then add the code in the following snippet
+to your app and call it in your code. This simple script initializes the Chat client, creates a chat room and sends a message, printing it to logcat when it is received over the websocket connection.
 
 ```kotlin
-val subscription = room.presence.subscribe { event: PresenceEvent ->
-    when (event.action) {
-        Action.enter -> println("${event.clientId} entered with data: ${event.data}")
-        Action.leave -> println("${event.clientId} left")
-        Action.update -> println("${event.clientId} updated with data: ${event.data}")
+import kotlinx.coroutines.delay
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.types.ClientOptions
```

Comment:
Snippet looks good, though import for `io.ably.lib.types.ClientOptions` clashes with `com.ably.chat.ClientOptions`.
Seems, we need to update class `com.ably.chat.ClientOptions` to either `com.ably.chat.ChatOptions` or something else

Reply:
Discussed in Slack and agreed a DR will follow - I'll update this as part of the DR